n3fit/src/n3fit/checks.py

```diff
@@ -459,6 +459,16 @@ def check_multireplica_qed(replicas, fiatlux):
     if fiatlux is not None:
         if len(replicas) > 1:
             raise CheckError("At the moment, running a multireplica QED fits is not allowed.")
+        
+@make_argcheck
+def check_photonQED_exists(theoryid, fiatlux):
+    """Check that the Photon QED set for this theoryid and luxset exists"""
+    if fiatlux is not None:
+      luxset = fiatlux['luxset']
+      try:
+          _ = FallbackLoader().check_photonQED(theoryid, luxset)
+      except FileNotFoundError:
+          raise CheckError(f"Photon QED set for TheoryID {theoryid.id} and luxset {luxset} not found.")
```

Comment:
```suggestion
      except FileNotFoundError as e:
          raise CheckError(f"Photon QED set for TheoryID {theoryid.id} and luxset {luxset} not found.") from e
```

I think until we manage to also make fiatlux faster we might want to still allow for the photon not to exist at the beginning of the fit so that it can be parallelized with the replicas.

Reply:
Well, fiatlux will run in the CPU and will be excruciatingly slow. But you can perfectly run it :')